.github/workflows/frontend-ci.yml

```diff
@@ -379,12 +373,6 @@ jobs:
       - name: Install package dependencies
         run: yarn install --frozen-lockfile --non-interactive
 
-      - name: 'DH: Start API'
-        if: ${{ env.is-main-branch == 'true' || needs.dh-affected.outputs.is-affected == 'true' }}
-        run: |
-          yarn nx run api-dh:ci-configuration
-          yarn nx serve api-dh:serve:ci
-
```

Comment:
question: Are these changes necessary in this PR?

Reply:
No that was a mistake, changes has now been reverted